### PR TITLE
Use RGBA textures for gradients and placeholders

### PR DESCRIPTION
**Pull request by DStradley**

### CODE REVIEW

**Review by pmconne — APPROVED** [review]

**Review by markschlosseratbentley — APPROVED** [review]

**Review by pmconne — COMMENTED** [review]
Run `rushchange.bat`.

File: core/common/src/Gradient.ts [hunk 1]
```diff
@@ -300,13 +300,11 @@ export namespace Gradient {
       const stepCount = Math.min(settings.stepCount, maxDimension);
       const dimension = (ThematicGradientMode.Smooth === settings.mode) ? maxDimension : stepCount;
       const hasAlpha = this.hasTranslucency;
```
Comment (pmconne) [inline core/common/src/Gradient.ts:302]: You could simplify things by getting rid of the `hasAlpha` check...if it is false then `color.getAlpha` below is going to return 255 for every color.

**Review by pmconne — APPROVED** [review]

File: core/common/src/Gradient.ts (hunk 1 above)
Comment (DStradley) [reply core/common/src/Gradient.ts]: Thanks, did not realize that.

**Comment by pmconne** [comment]
I don't understand why `rush change -v` is not complaining at you for having no change logs. Do you?

**Comment by DStradley** [comment]
> 

No I don't.  Forgot that step since it was a draft, should I do it now or let it run?

**Comment by pmconne** [comment]
The run has already gone past the check. Can you run `rushchange.bat` and see if it prompts you for a change log?
@calebmshafer does it seem weird to you as well that `rush change -v` succeeded despite @DStradley having committed no change logs?

**Pull request merged**